nni/algorithms/compression/v2/pytorch/pruning/basic_pruner.py

```diff
@@ -126,33 +127,33 @@ def compress(self) -> Tuple[Module, Dict]:
         return self.bound_model, masks
 
 
-class LevelPruner(BasicPruner):
+class _LevelPruner(BasicPruner):
     r"""
-    This is a basic pruner, and in some papers called it magnitude pruning or fine-grained pruning.
+    This is a basic pruner pruning the model in the granularity of single value. In some papers called it magnitude pruning or fine-grained pruning.
     It will mask the smallest magnitude weights in each specified layer by a saprsity ratio configured in the config list.
 
     Parameters
     ----------
-    model : torch.nn.Module
+    model
         Model to be pruned.
-    config_list : List[Dict]
+    config_list
         Supported keys:
             - sparsity : This is to specify the sparsity for each layer in this config to be compressed.
             - sparsity_per_layer : Equals to sparsity.
             - op_types : Operation types to be pruned.
             - op_names : Operation names to be pruned.
             - op_partial_names: Operation partial names to be pruned, will be autocompleted by NNI.
             - exclude : Set True then the layers setting by op_types and op_names will be excluded from pruning.
-    mode : str
+    mode
         'normal' or 'balance'.
-        If setting 'normal' mode, target tensor will be pruned in the way of finegrained pruning.
-        If setting 'balance' mode, a specal sparse pattern will chosen by pruner. Take linear
-        operation an example, weight tensor will be split into sub block whose shape is aligned to
-        balance_gran. Then finegrained pruning will be applied internal of sub block. This sparsity
-        pattern have more chance to achieve better trade-off between model performance and hardware
+        If set 'normal' mode, target tensor will be pruned in the way of finegrained pruning.
+        If set 'balance' mode, a special sparse pattern will be chosen by pruner. Take linear
+        operation as an example, weight tensor will be split into sub-block whose shape is aligned to
+        balance_gran. Then finegrained pruning will be applied internal of sub-block. This sparsity
+        pattern has more chance to achieve better trade-off between model performance and hardware
         acceleration. Please refer to releated paper for further information `Balanced Sparsity for
         Efficient DNN Inference on GPU <https://arxiv.org/pdf/1811.00206.pdf>`__.
-    balance_gran : list
+    balance_gran
```

Comment:
since this is a private pruner, some contents of docstring can be moved to the subclass, thus, avoid maintaining two copies of docstring, easily leading to inconsistent.

Reply:
removed